modules/cql/src/blaze/elm/concept.clj

```diff
@@ -0,0 +1,16 @@
+(ns blaze.elm.concept
```

Comment:
There should be a `concept-spec` namespace under `test` similar to the `code-spec` namespace. This `concept-spec` namespace should also be required in the `clinical-values-test`.